src/main/scala/com/advancedtelematic/campaigner/client/DirectorClient.scala

```diff
@@ -71,7 +73,7 @@ class DirectorHttpClient(uri: Uri)
       method = HttpMethods.PUT,
       uri    = uri.withPath(path),
     )
```

Comment:
i would add `.withNs(ns)` here, not when `execute` is called

Reply:
Indeed. Updated now.